config/sets/resolve-rdfjssource.json

```diff
@@ -0,0 +1,15 @@
+{
+  "@context": [
```

Comment:
Does this work properly?
You seem to be missing the `"https://linkedsoftwaredependencies.org/bundles/npm/ldflex-comunica/^2.0.0/components/context.jsonld"` context entry. (needed for expanding `config-sets-lc`)

Reply:
Ah yea, didn't change the context there. But yes, it actually worked, probably because it was still in memory from config-default?

Reply:
@rubensworks Is this a Components.js bug then?

Reply:
Probably a jsonld.js bug, I've seen things like this happening before...
It's not a big problem, but it should not occur anymore once we transition to my new JSON-LD parser :-)